program/src/processor.rs

```diff
@@ -579,8 +609,35 @@ pub fn withdraw_excess_inactive_sol<'a, 'b>(
     Ok(excess_balance)
 }
 
-/// Recover any inactive balance from the validator's stake accounts to the
-/// reserve account.
+pub fn get_stake_account_balance(
```

Comment:
This does more than just getting the balance, it also check the address. Maybe call it `check_address_and_get_balance`?

Reply:
Good idea to make this function though :+1: `withdraw_inactive_stake` is much easier to read now.